provider/pkg/schema/docs_schema.go

```diff
@@ -0,0 +1,49 @@
+package schema
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type Docs struct {
+	Types map[string]DocsTypes `json:"Types"`
+}
+
+type DocsTypes struct {
+	Properties  DocsProperties    `json:"properties"`
+	Attributes  map[string]string `json:"attributes"`
+	Description string            `json:"description"`
+}
+
+type DocsProperties map[string]string
+
+func ReadCloudFormationDocsFile(filePath string) (*Docs, error) {
+	docs := Docs{}
+	in, err := os.Open(filePath)
+	if err != nil {
+		return nil, fmt.Errorf("error opening file: %v", err)
+	}
+	defer in.Close()
+
+	dec := json.NewDecoder(in)
+	if err := dec.Decode(&docs); err != nil {
+		return nil, fmt.Errorf("error decoding: %v", err)
+	}
+	return &docs, nil
+}
+
+func (d *Docs) GetPropertyDesc(typeName, propName string) (string, bool) {
+	if typ, exists := d.Types[typeName]; exists {
+		if propName == "" {
```

Comment:
This is a little surprising compared to a separate `GetTypeDec(typeName) (string, bool)`

Reply:
What do you mean? Split this into two functions?

Reply:
Yeah, I would if it were a public API. This is just a stylistic nit, this would matter if we were building a library for pubilc use but internally I guess it just doesn't matter, feel free to disregard.